parser_library/test/context/instruction_test.cpp

```diff
@@ -0,0 +1,2500 @@
+{ "SLR", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 }, { system_architecture::YOP }, { system_architecture::ZOP }, { system_architecture::ESA }, { system_architecture::XA }, { system_architecture::_370 }, { system_architecture::DOS } } },
+{ "SLRK", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 } } },
+{ "SLXT", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 } } },
+{ "SLY", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 }, { system_architecture::YOP } } },
+{ "SORTL", { {system_architecture::UNI }, { system_architecture::Z15 } } },
+{ "SP", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 }, { system_architecture::YOP }, { system_architecture::ZOP }, { system_architecture::ESA }, { system_architecture::XA }, { system_architecture::_370 }, { system_architecture::DOS } } },
+{ "SPCTR", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 } } },
+{ "SPKA", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 }, { system_architecture::YOP }, { system_architecture::ZOP }, { system_architecture::ESA }, { system_architecture::XA }, { system_architecture::_370 }, { system_architecture::DOS } } },
+{ "SPM", { {system_architecture::UNI }, { system_architecture::Z15 }, { system_architecture::Z14 }, { system_architecture::Z13 }, { system_architecture::Z12 }, { system_architecture::Z11 }, { system_architecture::Z10 }, { system_architecture::Z9 }, { system_architecture::YOP }, { system_architecture::ZOP }, { system_architecture::ESA }, { system_architecture::XA }, { system_architecture::_370 }, { system_architecture::DOS } } },
+{ "SPT", { {system_architecture::UNI }, { system_architecture::Z15 }, { syste
```

Comment:
what does the TEST_F do here differently from TEST?

Reply:
I think the `TEST_F` is a leftover from some previous work where I needed access to diagnostics and using `TEST_F` with some addition functions was one of the ways to do this. Nevertheless, it is now `TEST`